src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -193,7 +194,7 @@ class TezosNodeOperator(val node: TezosRPCInterface)(implicit executionContext:
     */
   def getBlocksNotInDatabase(network: String, followFork: Boolean): Future[List[(Block, List[AccountId])]] =
     for {
-      maxLevel <- ApiOperations.fetchMaxLevel
+      maxLevel <- ApiOperations(dbHandle).fetchMaxLevel()
```

Comment:
Shouldn't this `ApiOperations`instance be memoized? The TezosNodeOperator will always refer to the same db... we could even pass the `apiOperations` instance as a dependency from the outside for the `TezosNodeOperator`